### PR TITLE
Add new Code::Stats plugin

### PR DESCRIPTION
**Pull request by p0358**
Could also someone accept my account on that x86 plugin page or tell me how other way I can upload x86 plugin? It didn't send me verify e-mail.

### CODE REVIEW

File: plugins/plugins64.xml [hunk 1]
```diff
@@ -81,6 +81,24 @@
...
+		<description>Code::Stats - Write code, level up, show off! A free stats tracking service for programmers. This plugin enables XP tracking in Notepad++.</description>
+		<author>p0358</author>
+		<homepage>https://github.com/p0358/notepadpp-CodeStats</homepage>
+		<sourceUrl>https://github.com/p0358/notepadpp-CodeStats</sourceUrl>
+		<latestUpdate>2017-02-30\n- Added the ability to change API URL in GUI\n- Added user-agent for API pulses\n- Some bug fixes</latestUpdate>
+		<stability>Good</stability>
+		<install>
+			<x64>
+				<download>https://github.com/p0358/notepadpp-CodeStats/releases/download/v1.0.1/notepadpp-CodeStats_x64.zip</download>
+				<copy from="CodeStats.dll" to="$PLUGINDIR$\" validate="true"/>
```
Comment (chcg) [inline plugins/plugins64.xml:98]: MD5 of CodeStats.dll needs to be added to validate.json
e.g.

```
    "JavaScript Map Parser":
    {
        "5f248d59e768a4702ab73f7cb6dc77e6": "JsMapParser.NppPlugin.dll"
    }
```,

**Comment by chcg** [comment]
Added https://npppm.bruderste.in/plugins/view/codestats.
@bruderstein Could you enable the access for p0358

File: plugins/plugins64.xml (hunk 1 above)
Comment (p0358) [reply plugins/plugins64.xml:98]: Added

**Comment by Nicd** [comment]
Don't know where else to report this, but I cannot seem to install the x86 version of this plugin. It looks like it is downloading for a moment, then it asks to restart N++ and after restart nothing has changed. I can install the plugin manually by extracting the DLL to the plugins folder. I have the latest version of N++ and the package manager.

**Comment by chcg** [comment]
@Nicd Thanks for reporting. The copy step configuration was missing for x86. It is available now on the dev list, but there is an expection on shutdown. 
@p0358 Are you aware of the N++ shutdown issue?

**Comment by p0358** [comment]
@chcg I think I am not. Can you explain further?

**Comment by chcg** [comment]
@p0358 See https://github.com/p0358/notepadpp-CodeStats/issues/2

**Pull request merged**